src/Custom-sample.h

```diff
@@ -345,75 +345,75 @@ static const char DATA_ESPEASY_DEFAULT_MIN_CSS[] PROGMEM = {
 // #define USES_P001   // Switch
 // #define USES_P002   // ADC
 // #define USES_P003   // Pulse
-// #define USES_P004   // Dallas
-// #define USES_P005   // DHT
-// #define USES_P006   // BMP085
+// #define USES_P004   // 1-Wire Temperature (Dallas/Maxim DS18B20)
+// #define USES_P005   // DHT11/12/22 SONOFF2301/7021/MS01
+// #define USES_P006   // BMP085/180
 // #define USES_P007   // PCF8591
-// #define USES_P008   // RFID
-// #define USES_P009   // MCP
+// #define USES_P008   // Wiegand (RFID)
+// #define USES_P009   // MCP23017
 
 // #define USES_P010   // BH1750
-// #define USES_P011   // PME
-// #define USES_P012   // LCD
-// #define USES_P013   // HCSR04
-// #define USES_P014   // SI7021
+// #define USES_P011   // ProMini Extender
+// #define USES_P012   // LCD2004
+// #define USES_P013   // HC-SR04/RCW-0001
+// #define USES_P014   // SI70xx/HTU21D
 // #define USES_P015   // TSL2561
 // #define USES_P017   // PN532
-// #define USES_P018   // Dust
+// #define USES_P018   // GP2Y10
 // #define USES_P019   // PCF8574
 
 // #define USES_P020   // Ser2Net
-// #define USES_P021   // Level
+// #define USES_P021   // Level Control
 // #define USES_P022   // PCA9685
-// #define USES_P023   // OLED
+// #define USES_P023   // OLED SSD1306
 // #define USES_P024   // MLX90614
 // #define USES_P025   // ADS1x15
 // #define USES_P026   // SysInfo
 // #define USES_P027   // INA219
-// #define USES_P028   // BME280
-// #define USES_P029   // Output
+// #define USES_P028   // BMx280
+// #define USES_P029   // Domoticz MQTT Helper
 
-// #define USES_P031   // SHT1X
-// #define USES_P032   // MS5611
-// #define USES_P033   // Dummy
+// #define USES_P031   // SHT1x
+// #define USES_P032   // MS5611 (GY-63)
+// #define USES_P033   // Dummy Device
 // #define USES_P034   // DHT12
-// #define USES_P036   // FrameOLED
+// #define USES_P036   // OLED SSD1306/SH1106 Framed
 // #define P036_FEATURE_DISPLAY_PREVIEW   1 // Enable Preview feature, shows on-display content on Devices overview page
 // #define P036_FEATURE_ALIGN_PREVIEW     1 // Enable center/right-align feature when preview is enabled (auto-disabled for 1M builds)
 // #define P036_ENABLE_TICKER   1 // Enable ticker function
-// #define USES_P037   // MQTTImport
+// #define USES_P037   // MQTT Import
 //   #define P037_MAPPING_SUPPORT 1 // Enable Value mapping support
 //   #define P037_FILTER_SUPPORT  1 // Enable filtering support
 //   #define P037_JSON_SUPPORT    1 // Enable Json support
 // #define USES_P038   // NeoPixel
-// #define USES_P039   // Environment - Thermocouple
+// #define USES_P039   // Thermosensors
```

Comment:
This one is specifically for thermocouple sensors, not just "Thermosensors" as that's too generic.

Reply:
All the "description" are taken from the official Device page https://espeasy.readthedocs.io/en/latest/Plugin/_Plugin.html

Reply:
Then maybe those could use an update too :)